src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -25,6 +28,12 @@
 
     private static final String FXML = "MainWindow.fxml";
 
+    // TODO: Add Lesson message usage once Eliana's part has been merged.
+    private static final String QUICK_START_INSTRUCTIONS = "💡 Quick Tips:\n"
+            + AddCommand.MESSAGE_USAGE + "\n\n"
+            + DeleteCommand.MESSAGE_USAGE + "\n\n"
+            + FindCommand.MESSAGE_USAGE;
+
```

Comment:
Will edit clear and list be in the quick start guide?

Reply:
I think `edit` can be done with delete and add so didn't add that in for quick tips. If we were adding `clear` and `list`, then it has like all the command usages already, would it be too much for a quick start cus the command result box is quite small?🤔🤔 I'm planning to add a summary table for all the commands for the help window separately

Reply:
@eeliana @lingshanng @angkoonhwee what do y'all think we should include in quick tips?

Reply:
I think clear should be in though since this is the first thing users see when they launch the application. So new users can learn the clear command and purge the list. @Xiaoyunnn

Reply:
actually I was thinking it can just be add and help, and maybe the language can sound more like talking to the user.
"You can add a person with the 'add' commannd ... " / "Try adding a person with..."
"You can find more commands with 'help' "

Reply:
@lingshanng @angkoonhwee i made some changes, see if they're ok?